python/mxnet/gluon/nn/basic_layers.py

```diff
@@ -657,26 +657,27 @@ def __repr__(self):
 
 
 class HybridLambda(HybridBlock):
-    r"""Wraps an operator or an expression as a HybridBlock object.
+    """Wraps an operator or an expression as a HybridBlock object.
```

Comment:
Can you comment on why you would remove this versus leave it in? I see it used a lot and I'm not sure if it is needed for the formatting that is below or if people just got used to using it all of the time.

Reply:
Issue "https://github.com/sphinx-doc/sphinx/issues/4479" suggests that a work around for most code block issues is adding a ```r"``` prefix to doc string. This makes this block a raw string. I thought we didn't want to consider the whole doc string as a raw string block unless there are too many things to escape in the string. 
http://openalea.gforge.inria.fr/doc/openalea/doc/_build/html/source/sphinx/rest_syntax.html#text-syntax-bold-italic-verbatim-and-special-characters

I could add it back if thats what is expected ..